src/DynamoCore/Library/TypedParameter.cs

```diff
@@ -42,7 +45,7 @@ public string Description
             get
             {
                 string description = string.Empty;
-                if (!string.IsNullOrEmpty(summary))
+                if (!string.IsNullOrEmpty(Summary))
```

Comment:
There is one regression now in master. Now we can't see parameter tooltips. 
I.e. that's what we see:
![image](https://cloud.githubusercontent.com/assets/8158404/8131002/f565d08a-111f-11e5-98b3-a30262a18fb2.png)

That's what we should see:
![image](https://cloud.githubusercontent.com/assets/8158404/8131032/21ec9260-1120-11e5-9394-5542676e3b6e.png)

I fixed this regression here.